golog.go

```diff
@@ -4,12 +4,19 @@ import (
 	"fmt"
 	"os"
 	"regexp"
+	"strings"
 	"time"
 
 	"github.com/codegangsta/cli"
 	"github.com/mitchellh/go-homedir"
 )
 
+const (
+	ExportICal = "ical"
```

Comment:
missing comment. All exported const should have a comment, (do we really need it exported? )